src/App.js

```diff
@@ -5,12 +5,16 @@ import { Reviews } from "./Components/Reviews";
 import { useEffect, useState } from "react";
 import { getReviews } from "./utils";
 import { FeaturedReviews } from "./Components/FeaturedReviews";
+import { SingleReview } from "./Components/SingleReview";
 
 function App() {
   const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Consider providing individual laoding state for each component fetching data from the api. Subsequently do your reviews need to be held in state within App.js at this stage